### DISCUSSION TITLE
[Slick Window Arrangement] Hotkey to disable snapping

### DISCUSSION THREAD

**Discussion by oSumAtrIX** [ramensoftware/windhawk-mods#3] [General]
Snapping is useful but can be annoying sometimes, which is why you might want to disable it occasionally. 
Photoshop for example lets you hold a key to disable snapping. This behaviour should be implemented to the mod `slick-window-arrangement`.

**Comment by m417z** [comment]
Thanks for the suggestion, that's a good idea. I'll look into implementing it.

**Comment by m417z** [comment]
Done in https://github.com/ramensoftware/windhawk-mods/pull/5, please update the mod and try it out.

**Reply by oSumAtrIX** [reply]
Thank you, this is a good addition. What could also be a good addition is to invert the conditions. Maby people only want snapping, when they press keys, opposed to the current way, when snapping is on by default and they have to press a key to disable it. Basically a setting could be added which inverts the behavior. Hold a key to enable snapping, stop pressing it to disable snapping again.
